numpy/core/tests/test_multiarray.py

```diff
@@ -5047,6 +5047,23 @@ def test_fromfile_offset(self):
                     np.fromfile, self.filename, dtype=self.dtype,
                     sep=",", offset=1)
 
+    def test_fromfile_bad_dup(self):
+        def dup_str(fd):
+            return 'abc'
+
+        def dup_bigint(fd):
+            return 2*68
```

Comment:
```suggestion
            return 2**68
```
I guess... Is that the reason why windows crashes hard?!